File: 백준/Bronze/4153. 직각삼각형/직각삼각형.js
let input = require("fs")
  .readFileSync("/dev/stdin")
  .toString()
  .trim()
  .split("\n");

let i = 0;
let option = true;

while (option) {
  let sides = input[i].split(" ").map(Number);

  if (sides[0] === 0 && sides[1] === 0 && sides[2] === 0) {
    option = false;
    break;
  }

  sides.sort((a, b) => a - b);

  if (sides[0] ** 2 + sides[1] ** 2 === sides[2] ** 2) {
    console.log("right");
  } else {
    console.log("wrong");
  }
  i++;
}

File: 백준/Bronze/4153. 직각삼각형/직각삼각형.kt
fun main() {
    val result = mutableListOf<String>()
    while (true) {
        val (a, b, c) = readln().split(" ").map { it.toInt() }.sorted()
        if (a == 0 && b == 0 && c == 0) break

        if (a * a + b * b == c * c) result.add("right") else result.add("wrong")
    }
    println(result.joinToString("\n"))
}